lib_xua/src/core/buffer/decouple/decouple.xc

```diff
@@ -726,6 +757,11 @@ void XUA_Buffer_Decouple(chanend c_mix_out
     t = array_to_xc_ptr(inZeroBuff);
     xc_ptr aud_to_host_zeros = t;
 
+    /* Stream format vars */
+    unsigned dataFormatOut, sampResOut;
```

Comment:
in general use int, it seems logical to use unsigned if you dont expect a value to be less than 0 but it can cause issues (unlikely to cause issues in this case, admittedly..)